RELEASE/scripts/autoscend/autoscend_header.ash

```diff
@@ -437,6 +437,234 @@ void awol_useStuff();
 effect awol_walkBuff();
 boolean awol_buySkills();
 
+########################################################################################################
+//Defined in autoscend/paths/bees_hate_you.ash
+boolean in_bhy();
+void bhy_initializeSettings();
+boolean bees_hate_usable(string str);
+boolean LM_bhy();
+boolean L13_bhy_towerFinal();
+
+########################################################################################################
+//Defined in autoscend/paths/casual.ash
+boolean inCasual();
+boolean inAftercore();
+boolean inPostRonin();
+boolean LM_canInteract();
+
+########################################################################################################
+//Defined in autoscend/paths/community_service.ash
+boolean LA_cs_communityService();
+boolean cs_witchess();
+void cs_initializeDay(int day);
+boolean do_chateauGoat();
+boolean cs_spendRests();
+void cs_make_stuff(int curQuest);
+boolean cs_eat_spleen();
+boolean cs_eat_stuff(int quest);
+void cs_dnaPotions();
+boolean cs_giant_growth();
+boolean auto_csHandleGrapes();
+int estimate_cs_questCost(int quest);
+int [int] get_cs_questList();
+int expected_next_cs_quest();
+string what_cs_quest(int quest);
+int expected_next_cs_quest_internal();
+boolean do_cs_quest(int quest);
+boolean do_cs_quest(string quest);
+int get_cs_questCost(int quest);
+int get_cs_questCost(string input);
+int get_cs_questNum(string input);
+void set_cs_questListFast(int[int] fast);
+boolean cs_preTurnStuff(int curQuest);
+boolean cs_healthMaintain();
+boolean cs_healthMaintain(int target);
+boolean cs_mpMaintain();
+boolean cs_mpMaintain(int target);
+boolean canTrySaberTrickMeteorShower();
+boolean trySaberTrickMeteorShower();
+int beachHeadTurnSavings(int quest);
+boolean tryBeachHeadBuff(int quest);
+
+########################################################################################################
+//Defined in autoscend/paths/dark_gyffte.ash
+void bat_startAscension();
+void bat_initializeSettings();
+boolean bat_wantHowl(location loc);
+boolean bat_formNone();
+boolean bat_formWolf(boolean speculative);
+boolean bat_formWolf();
```

Comment:
I'm not going to ask you to do this right now, in fact I'll probably do it myself, just noting here before I forget: The majority of the functions where we have two versions like this were just made because ASH didn't have default parameters yet. Now that it does I think I'll go through and switch to using default parameters where possible.

Reply:
Wait maybe it doesn't support default parameters? Weird, I could have sworn I saw a commit where they added that semi-recently, but I sure can't find that commit now, and they sure don't work.